pxtlib/blocks.ts

```diff
@@ -806,5 +808,20 @@ namespace pxt.blocks {
                 )
             })
         }
+
+        if (pxt.blocks.showBlockIdInTooltip) {
+            for (const id of Object.keys(_blockDefinitions)) {
+                const tooltip = _blockDefinitions[id].tooltip;
+                if (typeof tooltip === "object" && tooltip !== null) {
+                    for (const innerKey in tooltip) {
+                        if (tooltip.hasOwnProperty(innerKey)) {
```

Comment:
Not sure what this check will do.. If we're iterating through the `innerKey` of the specific tooltip, it should have that `innerKey`, right?

Reply:
It's to protect against any fields tooltip may have inherited which aren't directly on tooltip itself. I don't think it's strictly necessary here, since tooltip is defined pretty explicitly when it isn't a string, but I thought better safe than sorry.